src/main/java/appeng/util/Platform.java

```diff
@@ -30,7 +30,6 @@
 import java.util.List;
 import java.util.Random;
 import java.util.WeakHashMap;
-
 import javax.annotation.Nonnull;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 'javax.annotation.Nonnull' should be separated from previous imports. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)